port/common/omrportcontrol.c

```diff
@@ -284,6 +284,11 @@ omrport_control(struct OMRPortLibrary *portLibrary, const char *key, uintptr_t v
 		return 0;
 	}
 
+	if (0 == strcmp(OMRPORT_CTLDATA_VMEM_HUGE_PAGES_MMAP_ENABLED, key)) {
+		portLibrary->portGlobals->huge_pages_mmap_enabled = (BOOLEAN)value;
```

Comment:
there are two things here:
- you need to define macro name for `portLibrary->portGlobals->huge_pages_mmap_enabled` for consistency 
- I would avoid direct cast to `BOOLEAN` but allow key values `0` and `1` only
There is an example you can drill down:
```
	if (0 == strcmp(OMRPORT_CTLDATA_VMEM_PERFORM_FULL_MEMORY_SEARCH, key)) {
#if defined(PPG_performFullMemorySearch)
		Assert_PRT_true((0 == value) || (1 == value));
		PPG_performFullMemorySearch = value;
#endif /* PPG_performFullMemorySearch */
		return 0;
	}
......
#define PPG_performFullMemorySearch (portLibrary->portGlobals->platformGlobals.performFullMemorySearch)
.....
			if ((FALSE == error) && (0 == PPG_performFullMemorySearch)) {
```

Reply:
Yes, makes sense. Thanks for pointing this out